src/types/index.d.ts

```diff
@@ -457,6 +457,31 @@ declare module CuriefenseClient {
 
   // Operation documents - END
 
+  // User Details - START
+
+  type Key = {
```

Comment:
Key is a very very generic term, we should use a much more speicifc name for this type
```suggestion
  type UserAPIKey = {
```

Reply:
Done